src/test/java/org/json/junit/XMLTest.java

```diff
@@ -1239,7 +1239,8 @@ public void testIndentComplicatedJsonObjectWithArrayAndWithConfig(){
                     for (int numRead; (numRead = in.read(buffer, 0, buffer.length)) > 0; ) {
                         expected.append(buffer, 0, numRead);
                     }
-                    assertEquals(expected.toString(), actualString.replaceAll("\\n|\\r\\n", System.getProperty("line.separator")));
+                    assertEquals(expected.toString().replaceAll("\\n|\\r\\n", System.getProperty("line.separator")),
+                            actualString.replaceAll("\\n|\\r\\n", System.getProperty("line.separator")));
```

Comment:
Let's update these to use the more consise `System.lineSeparator()` now that it's supported instead of the longer `System.getProperty("line.separator")`.

Reply:
@johnjaylward good point, done.

(side note: not sure about this project's etiquette for code reviews, so I'm leaving this comment unresolved so you can review the change and decide yourself whether to resolve it or request further changes)

Reply:
I don't see an option to resolve the conversation. I believe only you (as the opener) or @stleary is able to resolve, so please resolve the discussion if you feel the changes have met the feedback request. If more changes are needed, a new review conversation can be started

![image](https://github.com/stleary/JSON-java/assets/383703/ac3b1c34-91d3-4587-8fdd-13c4123eec6b)